src/platforms/javascript/common/session-replay/understanding-sessions.mdx

```diff
@@ -69,3 +69,24 @@ await replay.flush();
 ```
 
 In `session` mode, this will upload any pending recording data to Sentry. In `buffer` mode, this will upload any pending recording data to Sentry and then continue recording, the same as when an error is sampled with `replaysOnErrorSampleRate`.
+
+## Custom Sampling
```

Comment:
I don't love this title -- thoughts @ryan953 ?

Reply:
hrm, 
Async Sampling?

It mentions "defer loading" right away... but i think that's the solution
the problem is:
> [...] they asked if there is a way to set the sample rate for session replays after Sentry is instantiated.

.... so wait a sec... idk if this is even the correct page for this doc.

This could be in the config/setup pages, because we're talking about setting the sample rate after the error monitoring has been enabled.

I just found: https://docs.sentry.io/platforms/javascript/session-replay/#lazy-loading-replay which has the core mechanism here too (totally fine to duplicate).

Reply:
What if we edited the file a little further up:
```diff
  ## Default Session Initialization

- By default, Replay will immediately start a session in either `session` or `buffer` mode, depending on your `replaysSessionSampleRate` and `replaysOnErrorSampleRate`.
+ By default, Replay will immediately start a session when the `new Replay()` integration is added to the SDK client. The session will be in either `session` or `buffer` mode, depending on your `replaysSessionSampleRate` and `replaysOnErrorSampleRate`.
```

and put this new section above the `Manually Starting Replay` section. Title this new one `Manually call addIntegration()` maybe? Or `Defer Adding Replay Integration`